src/plane.rs

```diff
@@ -168,6 +175,12 @@ impl<T: Pixel> Plane<T> {
     self.mut_slice(&PlaneOffset { x: 0, y: 0 })
   }
 
+  #[inline]
+  pub fn edged_slice(&self, po: &PlaneOffset, left_edge: usize, top_edge: usize) -> EdgedPlaneSlice<'_, T> {
```

Comment:
`left_edge` and `top_edge` could be anything different than 1.min(po.x) or 1.min(po.y) ?

Reply:
There are `go_left(3)` and `go_up(3)` with `PlaneSlice` elsewhere in the code base.

But I think the `.min(po.x)` could be done there.

Reply:
it would make the code even simpler :)

Reply:
I updated the PR :)